src/devices/Mcp25xxx/Mcp25xxx.cs

```diff
@@ -0,0 +1,407 @@
+        /// <param name="txb2">Instructs the controller to begin the message transmission sequence for TxB2.</param>
+        public void RequestToSend(bool txb0, bool txb1, bool txb2)
+        {
+            byte GetInstructionFormat()
+            {
+                byte value = 0x80;
+
+                if (txb0)
+                {
+                    value |= 0x01;
+                }
+
+                if (txb1)
+                {
+                    value |= 0x02;
+                }
+
+                if (txb2)
+                {
+                    value |= 0x04;
+                }
+
+                return value;
+            }
+
+            byte instructionFormat = GetInstructionFormat();
+            _spiDevice.WriteByte(instructionFormat);
+        }
+
+        /// <summary>
+        /// Quick polling command that reads several Status bits for transmit and receive functions.
+        /// </summary>
+        /// <returns>The response from READ STATUS instruction.</returns>
+        public ReadStatusResponse ReadStatus()
+        {
+            const byte instructionFormat = 0xA0;
+            const byte dontCare = 0x00;
+            ReadOnlySpan<byte> writeBuffer = stackalloc byte[] { instructionFormat, dontCare };
+            Span<byte> readBuffer = stackalloc byte[2];
+            _spiDevice.TransferFullDuplex(writeBuffer, readBuffer);
+            ReadStatusResponse readStatusResponse = (ReadStatusResponse)readBuffer[1];
+            return readStatusResponse;
+        }
+
+        /// <summary>
+        /// Quick polling command that indicates a filter match and message type
+        /// (standard, extended and/or remote) of the received message.
+        /// </summary>
+        /// <returns>Response from RX STATUS instruction.</returns>
+        public RxStatusResponse RxStatus()
+        {
+            const byte instructionFormat = 0xB0;
+            const byte dontCare = 0x00;
+            ReadOnlySpan<byte> writeBuffer = stackalloc byte[] { instructionFormat, dontCare };
+            Span<byte> readBuffer = stackalloc byte[2];
+            _spiDevice.TransferFullDuplex(writeBuffer, readBuffer);
+            RxStatusResponse rxStatusResponse = new RxStatusResponse(readBuffer[1]);
+            return rxStatusResponse;
+        }
+
+        /// <summary>
+        /// Allows the user to set or clear individual bits in a particular register.
+        /// Not all registers can be bit modified with this command.
+        /// </summary>
+        /// <param name="address">The address to write data.</param>
+        /// <param name="mask">The mask to determine which bits in the register will be allowed to change.
+        /// A '1' will allow a bit to change while a '0' will not.</param>
+        /// <param name="value">The value to be written.</param>
+        public void BitModify(Address address, byte mask, byte value)
+        {
+            const byte instructionFormat = 0x05;
```

Comment:
should instructionFormat be an enum?

Reply:
I will add a new InstructionFormat, but based on a base value.  It should be understood there are 3 instruction values that can change based on parameters.  Hopefully the included comments (along with datasheet) will help understand what is happening with related instruction formats.